tests/spread/core24-suites/scriptlets/scriptlet-failures/task.yaml

```diff
@@ -1,6 +1,7 @@
 summary: Validate scriptlet failures
 
-restore: |  # If this next line fails, delete it and replace it with the one following it
+restore:
+  | # If this next line fails, delete it and replace it with the one following it
```

Comment:
Can you drop this outdated comment and move the pipe symbol back to where it belongs?

This seems like a bug in the formatter.

Reply:
Now out of scope for this effort, a future enhancement will be to not ignore yaml files in test/spread

Reply:
Now out of scope for this effort, a future enhancement will be to not ignore yaml files in test/spread